docs/_posts/examples/3400-01-02-cluster.html

```diff
@@ -31,56 +30,55 @@
         clusterRadius: 50 // Radius of each cluster when clustering points (defaults to 50)
     });
 
-    // Use the earthquakes source to create five layers:
-    // One for unclustered points, three for each cluster category,
-    // and one for cluster labels.
     map.addLayer({
-        "id": "unclustered-points",
-        "type": "symbol",
-        "source": "earthquakes",
-        "filter": ["!has", "point_count"],
-        "layout": {
-            "icon-image": "marker-15"
+        id: "clusters",
+        type: "circle",
+        source: "earthquakes",
+        filter: ["has", "point_count"],
+        colorSpace: "lab",
+        paint: {
+            "circle-color": {
+                property: "point_count",
+                type: "interval",
+                stops: [
+                    [0, "#51bbd6"],
+                    [20, "#f1f075"],
+                    [150, "#f28cb1"],
+                ]
+            },
+            "circle-radius": {
+                property: "point_count",
+                type: "interval",
+                stops: [
+                    [0, 20],
+                    [1000, 40]
```

Comment:
color and radius should have the same number of stops with the same property values